operators/argo-controller/src/charm.py

```diff
@@ -53,7 +51,7 @@ def main(self, event):
 
         if not self.minio.is_available:
             self.log.info("Waiting for Minio data")
-            self.model.unit.status = MaintenanceStatus("Waiting for Minio data")
+            self.model.unit.status = WaitingStatus("Waiting for Minio data")
```

Comment:
Could this be updated to `Waiting for MinIO connection information` or similar? `data` is a little bit unclear.